prql-python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
```

Comment:
`cargo-extra-args = "--features pyo3/extension-module"` at the end of this file should be changed to `features = ["pyo3/extension-module"]`

See also https://www.maturin.rs/migration.html

Reply:
Thanks so much @messense !